client/app/bundles/course/material/folders/pages/FolderShow/index.tsx

```diff
@@ -37,6 +37,8 @@ const FolderShow: FC = () => {
   const { t } = useTranslation();
   const dispatch = useAppDispatch();
 
+  const data = useLoaderData();
+  if (!data) throw new Error('No download data. This should never happen.');
   // For new folder form dialog
```

Comment:
no need for these comments. Remove all of those

Reply:
Thats belong to previous commit, I think.

Reply:
I see. Then since you touch this file, might as well refactor it to make it prettier if it doesn't consume too much time.

Deleting comments definitely won't take too much time